src/js/utils/ui.js

```diff
@@ -0,0 +1,142 @@
+define([
+    'utils/backbone.events',
+    'events/events',
+    'utils/underscore'
+], function(Events, events, _) {
+    var TOUCH_MOVE = 'touchmove',
```

Comment:
I don't think declaring the names as vars helps with readability, but that's subjective. We aren't doing it elsewhere for 'click' so not sure we need to do it here.